Src/WitsmlExplorer.Frontend/services/jobService.tsx

```
@@ -91,6 +91,21 @@ export default class JobService {
       return [];
     }
   }
+
+  public static async cancelJob(
+    jobId: string,
+    abortSignal?: AbortSignal
+  ): Promise<Response> {
+    const response = await ApiClient.get(
+      `/api/jobs/cancel/${jobId}`,
+      abortSignal
+    );
+    if (response.ok) {
+      return response.json();
+    } else {
+      return null;
```

Instead of returning null if the response is not ok, use `throwError` to display an error message. You can look at `serverService.tsx` to see an example.

---

fixed